packages/language-chooser/src/style.scss

```diff
@@ -0,0 +1,141 @@
+/**
+ * Colors
+ */
+
+// WordPress grays.
+$black: #000; // Use only when you truly need pure black. For UI, use $gray-900.
+$gray-900: #1e1e1e;
+$gray-800: #2f2f2f;
+$gray-700: #757575;	// Meets 4.6:1 text contrast against white.
+$gray-600: #949494;	// Meets 3:1 UI or large text contrast against white.
+$gray-400: #ccc;
+$gray-300: #ddd;		// Used for most borders.
+$gray-200: #e0e0e0;	// Used sparingly for light borders.
+$gray-100: #f0f0f0;	// Used for light gray backgrounds.
+$white: #fff;
+
+.language-chooser__active-locales {
+	margin: 1em 0;
+
+	&::after {
+		content: "";
+		display: table;
+		clear: both;
+	}
+}
+
+.active-locales-controls .button {
+	margin: 0;
+}
+
+.language-chooser__active-locales-list,
+.language-chooser__inactive-locales-list {
+	width: 25em;
+	clear: both;
+	float: left;
+}
+
+.language-chooser__active-locales-list {
+	height: 130px;
+	overflow-y: scroll;
+	list-style: none;
+	margin: 0 1em 0 0;
+	padding: 0;
+	background: #fff;
+	box-shadow: inset 0 1px 2px rgba($black, 0.07);
+	border: 1px solid $gray-300;
+	box-sizing: border-box;
+}
+
+.language-chooser__inactive-locales-list {
+	margin: 0 1em 0 0;
+}
+
+.language-chooser__active-locales-list--empty {
+	background: rgba($white, 0.5);
+	border-color: rgba($gray-300, 0.75);
+	box-shadow: inset 0 1px 2px rgba($black, 0.04);
+	display: flex;
+	align-items: center;
+}
+
+.language-chooser__active-locales-list li {
```

Comment:
I'm a bit concerned if we're using consistent sizing, spacing and colors. Since I'm not a specialist, I think it would make sense to get a review from @WordPress/gutenberg-design here.